dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -270,17 +270,15 @@ def retain_header(header):
             header.find("7 days") > 0,
             # exclude "NAAT positivity rate - last 7 days - ages <5"
             header.find(" ages") < 0,
-        ]) or all([
+        ]) or (
             # include "Confirmed COVID-19 admissions - last 7 days"
-            header.startswith("Confirmed COVID-19 admissions"),
             # exclude "Confirmed COVID-19 admissions - percent change"
-            header.find("7 days") > 0,
             # exclude "Confirmed COVID-19 admissions - last 7 days - ages <18"
             # exclude "Confirmed COVID-19 admissions - last 7 days - age unknown"
-            header.find(" age") < 0,
             # exclude "Confirmed COVID-19 admissions per 100 inpatient beds - last 7 days"
-            header.find(" beds") < 0,
-        ])) or (all([
+            # exclude "Confirmed COVID-19 admissions per 100k - last 7 days"
+            header == "Confirmed COVID-19 admissions - last 7 days"
```

Comment:
I have confirmed through exhaustive checking that this is the only header ever used for the column we want, across all sheets and all dates up to 2022-02-24. A refreshing degree of consistency for this report!